main.js

```diff
@@ -1,4 +1,164 @@
-const h1 = document.querySelector('#h1');
-h1.addEventListener('click', () => {
-  h1.style.color = 'red';
-});
\ No newline at end of file
+const programInfo = [
+  {
+    id: 1,
+    img: 'img/icons/program_icon_01.png',
+    h2: 'Lecture',
+    p: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+  },
+  {
+    id: 2,
+    img: 'img/icons/program_icon_02.png',
+    h2: 'Exhibition',
+    p: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+  },
+  {
+    id: 3,
+    img: 'img/icons/program_icon_03.png',
+    h2: 'Forum',
+    p: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+  },
+  {
+    id: 4,
+    img: 'img/icons/program_icon_04.png',
+    h2: 'Workshop',
+    p: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+  },
+  {
+    id: 5,
+    img: 'img/icons/program_icon_05.png',
+    h2: 'Ignite',
+    p: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+  },
+];
+
+const program = document.querySelector('#programs');
+programInfo.forEach((e) => {
+  const programResume = document.createElement('div');
+  programResume.classList.add('progResume');
+  programResume.setAttribute('id', e.id);
+  program.appendChild(programResume);
+
+  const programImg = document.createElement('img');
+  programImg.classList.add('flexItem1');
+  programImg.classList.add('progImgSize');
+  programImg.setAttribute('src', e.img);
+  programResume.appendChild(programImg);
+
+  const programH2 = document.createElement('h2');
+
+  programH2.classList.add('progH2');
+  programH2.classList.add('flexItem1');
+  programH2.textContent = e.h2;
+  programResume.appendChild(programH2);
+
+  const programP = document.createElement('p');
+
+  programP.classList.add('progP');
+  programP.classList.add('flexItem2');
+  programP.textContent = e.p;
+  programResume.appendChild(programP);
+});
+
+const expositorInfo = [
+  {
+    id: 'speaker1',
+    img: 'img/speakers/speaker_01.png',
+    h2: 'Yochai Benkler',
+    h3: 'Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School',
+    p: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+  },
+  {
+    id: 'speaker2',
+    img: 'img/speakers/speaker_02.png',
+    h2: 'Kilnam Chon',
+    h3: '',
+    p: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Tellus orci ac auctor augue.',
```

Comment:
- [OPTIONAL] I think it's better to avoid long lines of code. Makes your code more readable and professional. As not required by the previous reviewer I consider this optional.